fast64_internal/mk64/f3d/properties.py

```diff
@@ -0,0 +1,45 @@
+from bpy.props import StringProperty, EnumProperty, FloatProperty, BoolProperty
+from bpy.types import PropertyGroup, UILayout
+from bpy.utils import register_class, unregister_class
+from ...utility import prop_split
+from ...render_settings import (
+    on_update_render_settings,
+)
+from ...f3d.f3d_material import ootEnumDrawLayers
+
+
+class MK64CourseDLImportSettings(PropertyGroup):
+    DLImportName: StringProperty(name="Name")
+    DLImportPath: StringProperty(name="Directory", subtype="FILE_PATH")
+    DLImportBasePath: StringProperty(name="Directory", subtype="FILE_PATH")
+    blenderF3DScale: FloatProperty(name="F3D Blender Scale", default=100, update=on_update_render_settings)
```

Comment:
ideally you'd make a scale variable for mk64 overall like sm64 and oot instead of making it specfic to these import properties

Reply:
hmm what do you mean exactly because in oot I only see an autoscale for actor

Reply:
Basically game modes have a blender to game scale,  if mk64 ever implements exporting or any other functionality that would need a scale factor it would be nice to only have one